Jenkinsfile

```diff
@@ -279,6 +280,14 @@ if (env.CHANGE_TITLE && env.CHANGE_TITLE.contains('[ci-skip]')) {
                                                 checkout scm
                                             }
 
+                                        println "Test React Native"
+                                        dir("react-native") {
+                                            sh "npm i"
+                                            sh "npm run gobuild-ios"
+                                            sh "(npm run start &) ; npm run test-ios"
+                                            sh "killall 'Simulator'"
```

Comment:
adding in parallel now. @chromakode doesn't seem like there is a better way. this is what stackoverflow suggests and it works